tests/operators/test_fast_fourier_op.py

```diff
@@ -101,3 +101,17 @@ def test_fast_fourier_op_onematrix():
         FastFourierOp(recon_matrix=recon_matrix, encoding_matrix=None)
     with pytest.raises(ValueError, match='None'):
         FastFourierOp(recon_matrix=None, encoding_matrix=encoding_matrix)
+
+
+def test_invalid_dim():
+    """Test that the FastFourierOp throws an error if not all dimensions
```

Comment:
Tests that dims are in (-3,-2,-1) if recon_matrix or encoding_matrix is SpatialDimension